src/simulation/sensors/imuSensor/imuSensor.cpp

```diff
@@ -416,3 +410,79 @@ void ImuSensor::UpdateState(uint64_t CurrentSimNanos)
 
     return;
 }
+
+/*!
+    Setter for `AMatrixAccel`
+    @param propMatrix Matrix to set
+*/
+void ImuSensor::setAMatrixAccel(const Eigen::MatrixXd& propMatrix)
+{
+    this->AMatrixAccel = propMatrix;
+    this->errorModelAccel.setPropMatrix(propMatrix);
+}
+
+/*!
+    Setter for `AMatrixGyro`
+    @param propMatrix Matrix to set
+*/
+void ImuSensor::setAMatrixGyro(const Eigen::MatrixXd& propMatrix)
+{
+    this->AMatrixGyro = propMatrix;
+    this->errorModelGyro.setPropMatrix(propMatrix);
+}
+
+/*!
+    Getter for `AMatrixAccel`
+    @return Current matrix
+*/
+Eigen::MatrixXd ImuSensor::getAMatrixAccel() const
+{
+    return this->AMatrixAccel;
+}
+
+/*!
+    Getter for `AMatrixGyro`
+    @return Current matrix
+*/
+Eigen::MatrixXd ImuSensor::getAMatrixGyro() const
+{
+    return this->AMatrixGyro;
+}
+
+/*!
+    Setter for `walkBoundsAccel`
+    @param bounds Bounds vector to set
+*/
+void ImuSensor::setWalkBoundsAccel(const Eigen::Vector3d& bounds)
+{
+    this->walkBoundsAccel = bounds;
+    this->errorModelAccel.setUpperBounds(bounds);
+}
+
+/*!
+    Setter for `walkBoundsGyro`
+    @param bounds Bounds vector to set
+*/
+void ImuSensor::setWalkBoundsGyro(const Eigen::Vector3d& bounds)
+{
+    this->walkBoundsGyro = bounds;
+    this->errorModelGyro.setUpperBounds(bounds);
+}
+
+/*!
+    Getter for `walkBoundsAccel`
+    @return Current bounds
+*/
+Eigen::Vector3d ImuSensor::getWalkBoundsAccel() const
+{
+    return this->walkBoundsAccel;
+}
+
+/*!
+    Getter for `walkBoundsGyro`
+    @return Current bounds
+*/
+Eigen::Vector3d ImuSensor::getWalkBoundsGyro() const
+{
+    return this->walkBoundsGyro;
+}
```

Comment:
Update the imuSensor.rst file to discuss how to properly set the noise values.  You can note that the use of directly setting `walkBoundsAccel` is now deprecated.